src/script/conversation/ConversationRepository.ts

```diff
@@ -616,27 +617,36 @@ export class ConversationRepository {
   ): Promise<ContentMessage[]> {
     const hasAdditionalMessages = events.length === Config.getConfig().MESSAGES_FETCH_LIMIT;
 
+    const isMemberMessage = (
```

Comment:
thats a typeguard that is only used inside this function.

Do we have a place where typegards can live? If not we may need to create one (a folder) because we need to implement typeguards more often in the future.